### PR TITLE
donate form 

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: theme/js/script.js [hunk 1]
```diff
@@ -44,46 +44,53 @@ $(document).ready(function() {
 
     countCharacter();
 
+    openDonationContent();
```
Comment (ElXyfer) [inline theme/js/script.js]: You don’t need this line, you can remove it.

File: theme/js/script.js [hunk 2]
```diff
@@ -44,46 +44,53 @@ $(document).ready(function() {
...
+
     /* Calling the clickableElement function on card-wrapper.
     This function needs parameters running through it to work. */
     clickableElement($('.card-component-container .card-wrapper'));
 
+    document.getElementById("defaultOpen").click();
```
Comment (ElXyfer) [inline theme/js/script.js]: You don’t need this line, you can remove it.

File: theme/js/script.js (hunk 1 above)
Comment (katlazarova) [reply theme/js/script.js]: done

File: theme/js/script.js (hunk 2 above)
Comment (katlazarova) [reply theme/js/script.js]: done

File: theme/js/script.js [hunk 3]
```diff
@@ -176,22 +182,97 @@ $.getUrlParameter = function(name){
...
+        $('.donate-form-buttons .donate-sum').not('.donate-sum.active').removeClass('active');
+    });
+
+    // Clicking the donate button redirects to a thank you page.
+    $('.tabcontent .donate-button').click(function () {
+        var donationSum = $('.tabcontent.active .donate-sum.active').text();
+        var donationSumOther = $('.tabcontent.active .donate-form-input.active').val();
+
+        if (donationSum == null) {
+            window.location.href = '/thank-you-page.php?donationSum=' + donationSum + '';
```
Comment (ElXyfer) [inline theme/js/script.js]: You, don't need the `+ ''` at the end Because you're just adding an empty string. Same for line 244

Comment (katlazarova) [reply theme/js/script.js]: done

Comment (katlazarova) [reply theme/js/script.js]: this breaks the js.

File: theme/js/script.js [hunk 4]
```diff
@@ -48,42 +48,69 @@ $(document).ready(function() {
     This function needs parameters running through it to work. */
     clickableElement($('.card-component-container .card-wrapper'));
 
+    // openDonationText(event, 25);
```
Comment (ElXyfer) [inline theme/js/script.js]: Remove this.

File: thank-you-page.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?php include 'includes/header.php'; ?>
+<h1>SUCCESS!</h1>
+<div class="thank-you-message">
+    <h2>Thank you for your support!</h2>
+    <div class="donation-success-message">
+    </div> <!-- End of donation-success-message -->
+</div> <!-- End of thank-you-message -->
+<div class="link-to-blog">
+    <p>Find out about the work we're doing</p>
+<a href="/blog.php">here</a>
```
Comment (ElXyfer) [inline thank-you-page.php]: Add a full stop after anchor tag

File: theme/js/script.js [hunk 6]
```diff
@@ -176,22 +204,104 @@ $.getUrlParameter = function(name){
...
+        // sets the initial position of the arrow to equal the middle of the first button.
+        $('.donation-content-panel .triangle-left').css({left: initialPosition.left + buffer});
+    };
+
+    setInitialPosition();
+
+    $('.tab .tablinks').click(function () {
+        //Clicking on tablinks adds an active class and removes it from the sibling element.
+        $(this).addClass('active').siblings().removeClass('active');
+        if ($(this).hasClass('monthly')) {
```
Comment (ElXyfer) [inline theme/js/script.js]: This section can be refactored give me a shout for guidance.

File: thank-you-page.php (hunk 5 above)
Comment (katlazarova) [reply thank-you-page.php]: done

File: includes/header.php [hunk 7]
```diff
@@ -29,7 +29,7 @@
                 </button>
                 <ul class="nav-list">
                     <li>
-                        <a href="/">Home</a>
+                        <a href="/index.php">Home</a>
```
Comment (katlazarova) [inline includes/header.php]: revert this change

File: theme/js/script.js [hunk 8]
```diff
@@ -176,22 +202,111 @@ $.getUrlParameter = function(name){
...
+            var mainAmount = 'monthly';
+            var otherAmount = 'single';
+            tabFunctionality(mainAmount, otherAmount);
+            setInitialPosition();
+            // Add active class to the first donate form monthly text and remove it from the siblings.
+            $('.donate-form-text-monthly .donate-form-text').first().addClass('active').siblings().removeClass('active');
```
Comment (katlazarova) [inline theme/js/script.js]: add this line and the one below into tabfunctionality function

File: theme/js/script.js [hunk 9]
```diff
@@ -176,22 +202,111 @@ $.getUrlParameter = function(name){
...
+            $('.donate-form-text-monthly .donate-form-text').removeClass('active');
+        }
+    });
+
+    $('.donate-form-buttons .donate-sum').click(function () {
+        // When a donate-sum button is clicked, add an active class and remove it from the siblings.
+        $(this).addClass('active').siblings().removeClass('active');
+
+        var buttonPosition = $('.donate-form-buttons .donate-sum.active').index();
+        var targetTextMonthly = $('.donate-form-text-monthly .donate-form-text') [buttonPosition];
```
Comment (katlazarova) [inline theme/js/script.js]: remove space before square bracket on this line and the one below

File: theme/js/script.js [hunk 10]
```diff
@@ -176,22 +202,111 @@ $.getUrlParameter = function(name){
 };
 
 function countCharacter() {
-    $('.text-area').on('input', function(){
+    $('.text-area').on('input', function () {
```
Comment (katlazarova) [inline theme/js/script.js]: make selector more specific

File: theme/js/script.js (hunk 6 above)
Comment (katlazarova) [reply theme/js/script.js]: done

File: includes/header.php (hunk 7 above)
Comment (katlazarova) [reply includes/header.php]: done

File: theme/js/script.js (hunk 8 above)
Comment (katlazarova) [reply theme/js/script.js]: done

File: theme/js/script.js (hunk 9 above)
Comment (katlazarova) [reply theme/js/script.js]: done

File: theme/js/script.js (hunk 10 above)
Comment (katlazarova) [reply theme/js/script.js]: done
